package.json

```diff
@@ -16,8 +16,8 @@
   "devDependencies": {
     "chai": "^4.1.2",
     "jshint": "^2.9.5",
-    "mocha": "^5.0.4",
-    "mock-require": "^3.0.1",
+    "mocha": "^5.0.5",
+    "mock-require": "^3.0.2",
```

Comment:
These upgraded development dependencies do not have any functional/behavioural impact on the app since they are not transitive.